src/libs/workflow-utils.ts

```diff
@@ -53,13 +54,15 @@ export const downloadIO = (io, filename) => {
   FileSaver.saveAs(blob, `${filename}.json`);
 };
 
-export const downloadWorkflows = (rows, filename) => {
+export const downloadWorkflows = (rows: Workflow[], filename: any) => {
```

Comment:
```suggestion
export const downloadWorkflows = (rows: Workflow[], filename: string) => {
```
Maybe make the filename a string.

Reply:
Thanks for the suggestions! I'll add these